### ISSUE TITLE
How can I debug keycastr?

### ISSUE THREAD

**Issue by lqez** [keycastr/keycastr#114]
To run keycastr, I have to add new binary into Accessibility/Privacy panel every time. So I cannot debug in XCode.
How can I run keycastr with debug run in XCode?

**Comment by akitchen** [comment]
Trying bringing Xcode into that privacy panel instead (or in addition)? 👍
